Upgrade branch 2.7.x using TemplateControl

octonato opened the pull request:
```
Updated with template-control on 2019-01-10T23:13:52.939Z
  /LICENSE:
    wrote /LICENSE
  /NOTICE:
    wrote /NOTICE
  /.travis.yml:
    wrote /.travis.yml
  **/plugins.sbt:
    addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.0-RC9")
  **build.gradle:
    def playVersion = "2.7.0-RC9"
  **build.gradle:
        playTest "org.scalatestplus.play:scalatestplus-play_\$scalaVersion:4.0.0-RC2"

```

marcospereira commented on build.gradle:
```diff
@@ -25,7 +25,7 @@ dependencies {
     play "com.typesafe.play:play-logback_$scalaVersion:$playVersion"
     play "com.typesafe.play:filters-helpers_$scalaVersion:$playVersion"
-    playTest "org.scalatestplus.play:scalatestplus-play_$scalaVersion:4.0.0-RC2"
+    playTest "org.scalatestplus.play:scalatestplus-play_\$scalaVersion:4.0.0-RC2"
```
I'm blocking merge on this PR as a reminder that this replacement is not correct. We should only use this scaping slash on the seeds projects.

octonato replied:
I resend it. The seed will get broken tough

octonato committed:
Updated with template-control on 2019-01-10T23:19:30.867Z
  /LICENSE:
    wrote /LICENSE
  /NOTICE:
    wrote /NOTICE
  /.travis.yml:
    wrote /.travis.yml
  **/plugins.sbt:
    addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.0-RC9")
  **build.gradle:
    def playVersion = "2.7.0-RC9"

octonato committed:
back to 'flashing'
